### PR TITLE
Get Wazuh version from VERSION file

### LINKED ISSUE #102: Missing tools and files in packages
## Description

Related issue: https://github.com/wazuh/wazuh-indexer/issues/99

Comparing 4.9.0 packages with packages from a prior version, we found out that these tools are missing in our packages.

```
/usr/share/wazuh-indexer/plugins/opensearch-security/tools/config.yml	                removed
/usr/share/wazuh-indexer/plugins/opensearch-security/tools/wazuh-passwords-tool.sh	removed		
/usr/share/wazuh-indexer/plugins/opensearch-security/tools/wazuh-certs-tool.sh	        removed		
```

We need these files back.

- https://packages.wazuh.com/4.7/config.yml
- https://packages.wazuh.com/4.7/wazuh-certs-tool.sh
- https://packages.wazuh.com/4.7/wazuh-passwords-tool.sh

## Tasks

- Extend the packages' `assemble` tool to download and install the files above to their expected path. 

## Implementation restrictions
- Preserve versioning.

### PR DESCRIPTION
**Pull request by f-galland**
### Description
This PR fixes getting the Wazuh Indexer version from the `VERSION` file within the repository when downloading Wazuh-specific configuration files and scripts.

### Issues Resolved
Resolves #102 

### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
- [x] Commits are signed per the DCO using --signoff
- [ ] Commit changes are listed out in CHANGELOG.md file (See: [Changelog](../blob/main/CONTRIBUTING.md#changelog))
- [ ] GitHub issue/PR created in [OpenSearch documentation repo](https://github.com/opensearch-project/documentation-website) for the required public documentation changes (#[Issue/PR number])

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by f-galland** [comment]
Packages are correctly generated with the right version string using the dockerized build and assemble environments:

DeepinScreenshot_select-area_20240115161752

**Review by AlexRuiz7 — CHANGES_REQUESTED** [review]
Let's try to split the new Docker environments to a new issue and PR.

File: scripts/provision.sh [hunk 1]
```diff
@@ -3,7 +3,7 @@
 # Provision script for assembly of DEB packages
 
 # Install necessary packages
-apt-get update -y && apt-get upgrade -y && apt-get install -y curl build-essential curl &&
+apt-get update -y && apt-get upgrade -y && apt-get install -y curl wget build-essential &&
```
[the remark this answers is not in the document]
Comment (f-galland) [reply scripts/provision.sh]: Fixed.

**Review by AlexRuiz7 — APPROVED** [review]
LGTM!

Workflow runs successfully! 

https://github.com/wazuh/wazuh-indexer/actions/runs/7582882220

**Pull request merged**